.ci/travis-functions.sh

```diff
@@ -65,21 +64,18 @@ function xdebug_enable() {
 #   String
 #######################################
 function get_infection_pr_flags() {
-    local flags="--no-progress";
+    local flags="";
     local changed_files;
-    local min_msi;
 
     if ! [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
         git remote set-branches --add origin "$TRAVIS_BRANCH";
         git fetch;
 
         changed_files=$(git diff origin/"$TRAVIS_BRANCH" --diff-filter=A --name-only | grep src/ | paste -sd "," -);
-        min_msi=$(($(grep -o 'min-msi=[0-9]*' .travis.yml | head | cut -f2 -d=) + 1))
```

Comment:
Is there a reason to _bump_ the MSI score in PRs?

Reply:
why not? to force increasing MSI

Reply:
That's cool if it's a PR with new changes, otherwise when it's refactoring something where there is nothing to improve it's gonna be a red build for nothing.

On a side note I find way more damaging to have non-actionable build failures (e.g. Codecov) because it makes you complacent towards red builds

Reply:
We have a `--diff-filter=A` that means only added files are counted for MSI checks, so refactoring of the existing files should not fail the build.

Reply:
More than once though git understood a file as bien added instead of moved though; maybe that's why I got that impression a lot